maps/chronosphere/main.lua

```diff
@@ -1,15 +1,17 @@
 -- chronosphere --
 
 require "modules.difficulty_vote"
+require "maps.chronosphere.blueprints_vote"
 require "modules.biters_yield_coins"
 require "modules.no_deconstruction_of_neutral_entities"
 --require "modules.no_solar"
 require "modules.shotgun_buff"
-require "modules.mineable_wreckage_yields_scrap"
 require "maps.chronosphere.comfylatron"
 require "maps.chronosphere.terrain"
+require "maps.chronosphere.weapon_buffs"

```

Comment:
weapon buffs remove as whole, shotgun is already op and pistol can be rebalanced in its own module. ALso having pistol being strong is THE POINT of it

Reply:
I think having the pistol being medium-strong is more interesting balance-wise.